src/main/java/com/jcraft/jsch/Session.java

```diff
@@ -73,7 +73,7 @@
   private static final int PACKET_MAX_SIZE = 256 * 1024;
 
   private byte[] V_S;                                 // server version
-  private byte[] V_C=Util.str2byte("SSH-2.0-JSCH_"+Version.getVersion()); // client version
+  private byte[] V_C=Util.str2byte("SSH-2.0-JSCH_"+JSch.VERSION); // client version
```

Comment:
This part needs to be reverted I think.

Reply:
It's unrelated to the logging issue but worth thinking about the reason why the "logic" to get the implementation's version needs to be done twice.

Reply:
This probably could remain. Calling `Version.getVersion()` here originated because earlier I mistakenly removed `JSch.VERSION`, but then re-added it to restore backwards compatibility in #98.